src/annotator/config/test/settings-test.js

```diff
@@ -88,10 +113,31 @@ describe('annotator.config.settingsFrom', function() {
       location: {
         href: href,
       },
+      document: {
+        querySelector: sinon.stub().returns({href: 'hi'}),
```

Comment:
As a reader, I find it helpful for understanding if fake data in tests matches the type of the real data. eg. Returning a URL rather than an arbitrary string here, even if the code being tested doesn't actually care.